api.yaml

```diff
@@ -168,6 +168,108 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProjectListResponse'
+    post:
+      operationId: create-project
+      tags:
+        - projects
+      summary: Create new project.
+      description: |
+        This method is used to create a new project through API.
+      externalDocs:
+        url: 'https://developers.qase.io/#create-a-new-project'
```

Comment:
nope, please do not add external docs.

motivation: this source will be a point of true, and replace the origin